src/meson.build

```diff
@@ -0,0 +1,67 @@
+incdir = include_directories('../include/')
+
+install_headers(
+    '../include/lsst/sphgeom/Angle.h',
+    '../include/lsst/sphgeom/AngleInterval.h',
```

Comment:
Personally, I would define this in `subdir('include')` so you don't need all these ../ everywhere. :)